tools/iana_to_rflx.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env -S python3 -O
+
```

Comment:
There should be an easy way to regenerate the RecordFlux specifications from the XML files. The generated specifications should be verified after generation (this could be also generally done in the tool). The CI should ensure that the XML files and the generated files are in sync.

Reply:
> There should be an easy way to regenerate the RecordFlux specifications from the XML files.

I added a make targets for that purpose in 5812afdd

> The generated specifications should be verified after generation (this could be also generally done in the tool)

Done in 8a3fe271

> The CI should ensure that the XML files and the generated files are in sync.

So there should be a test-case for that?

Reply:
> > The CI should ensure that the XML files and the generated files are in sync.
> 
> So there should be a test-case for that?

Yes, that would be an option.

Reply:
Added a test case in 2b2c077.